aliases/available/thefuck.aliases.bash

```diff
@@ -0,0 +1,5 @@
+# https://github.com/nvbn/thefuck
+alias fuck='eval $(thefuck $(fc -ln -1))'
+alias please='fuck'
+alias plz=please
+alias fucking=fuck
```

Comment:
As per previous note regarding `fucking` not being a good alias for "try this command again" I think it would be better as a simple alias for `sudo`, even here in this file.

So fo consistency across the two alias files, I think we do one of:

1) Keep it in both files as an alias for `sudo`
2) Remove it from both files

I think my vote is for # 1 since that alias is kind of the reason this PR was created in the first place.